### ISSUE TITLE
_migrations table not replicated

### ISSUE THREAD

**Issue by jeffreywescott** [JohanObrink/rethink-migrate#19]
rethink-migrate seems to have no way to set the `replicas` attribute when creating the `_migrations` table ([see here](https://rethinkdb.com/api/javascript/table_create/#description)), which means that in a real-world production environment, the `_migrations` table is not being replicated across all nodes in the cluster.

**Comment by jordanh** [comment]
This is a little meta, but one could write a migration for the `_migrations` table to tag replication for this table, no?

**Comment by jeffreywescott** [comment]
Interesting idea, @jordanh ... still, I think it would be better as a feature on this module.

**Comment by jordanh** [comment]
What would you propose as the interface? Configuration options in `database.json` and/or command line switches?

**Comment by jeffreywescott** [comment]
Yeah, I think something like `numReplicas` and `numShards` in `database.json` is the way to go.
